docs/resources/widgets/pages/pages.md

```diff
@@ -0,0 +1,142 @@
+**Header** (`AppBar`), **Body**, **Floating Action Button, Drawer,** and other typical page
+components. In FlutterFlow, you will find these page widgets under **Page
+Elements**.
+
+:::tip[Generated Code]
+
+[See how](../generated-code.md) FlutterFlow converts your page into Flutter
+code
+:::
+
+Here's how the `Scaffold` contributes to page design in FlutterFlow:
+
+**AppBar** : Scaffold allows you to easily include an `AppBar` at the top of the
+page,
+which can house the title, navigation controls, and other actions.
+
+**Floating Action Button (FAB)**: An action button that is commonly used for
+primary actions on the screen, like adding a new contact or composing a message.
+
+**Drawer & End-Drawer**: A slide-out menu for app navigation, accessible from
+the `AppBar` or by swiping from the side.
+
+**Body:** The main content area where you place your widgets. In FlutterFlow,
+you
+won't find a section explicitly labeled "Body". However, the widget tree that
+you assemble under your "PageNameWidget" essentially functions as the body of the
+page. This is where you add and organize all the interactive and static elements
+that make up the user interface of that particular page.
+
+![body-page.png](..%2Fimgs%2Fbody-page.png)
+
+## Navigation / Routing
+
+Navigation in FlutterFlow involves moving between different pages. This is
+handled through routing, where each page is identified by a unique route.
+Navigation can be programmed to happen on events like button clicks, leading to
+actions such as pushing a new route (opening a new page) or popping a route (
+returning to a previous page). FlutterFlow simplifies the routing process,
+allowing you to visually design the navigation flow of your app.
+
+Learn more about the [navigation actions](how-tos.md#navigation) in FlutterFlow.
+
+## Page Parameters
+
+Parameters in FlutterFlow are used to pass data between pages. When navigating
+from one page to another, you can send parameters to configure the destination
+page based on the data from the current page. This is useful for tasks like
+passing a user ID to a profile page or specific details to a detailed view page.
+
+To manage creation and updation of Page parameters, [see here](how-tos.md#page-parameters)
+
+## Page state
+
+**Page state** refers to the information that a page tracks about its current
+condition or the data it displays. This can include things like which tabs are
+active, the contents of a form, or any user interactions. Managing state is
+crucial for dynamic pages that interact with user inputs or load varying data.
+
+To manage creation and updation of Page State variables, [see here](how-tos.md#page-state)
+
+## Page Lifecycle
+
+In FlutterFlow and Flutter, understanding the page lifecycle is crucial as it
+involves the stages a page goes through from creation to disposal. Recognizing
+these stages is essential for managing resources and data effectively.
+
+Let's delve into the key moments in the page lifecycle:
+
```

Comment:
I'm a bit confused as to what to make of this section. What do we want the user to take away from here? 

I think we probably want the user to understand when a page is reloaded (could be from the code or user interaction like refreshing a page) and what happens when its reloaded (i.e. page state variables are reset to their initial value) -- I feel like we're not providing enough helpful info here